website/docs/reference/manifest.md

```diff
@@ -209,6 +209,15 @@ Keys are human-readable link names, and values are URLs.
 "We're hiring" = "https://swmansion.com/careers/"
 ```
 
+### `experiments`
+
+This field is responsible for setting experimental flags to be used on the package for the compiler.
+
+```toml
+[package]
+experiments = ["negative_impls"]
+```
+
```

Comment:
Please create an issue to include a list of all supported experimental features in our documentation. I'd like this list to be auto-generated from code, so that won't be trivial so let's do this as separate chunk of work

Reply:
#1041